src/app_service/service/community/service.nim

```diff
@@ -240,6 +240,10 @@ QtObject:
       if (chat.categoryId == categoryId):
         let fullChatId = community.id & chat.id
         var chatDetails = self.chatService.getChatById(fullChatId)
+        # chats moved to a new category must be searched by chat.id
```

Comment:
Interesting. Can you give an example of community.id and chat.id. I am wondering why chat's id has changed in the service.